src/app/visningskomponenter/personalia/barn.tsx

```diff
@@ -34,7 +34,7 @@ function Barn(props: Pick<PersonaliaInfo, 'barn'>) {
 
     const { barn, ...rest} = props;
 
-    const barnListe = barn.map((ettBarn) => <EnkeltBarn barn={ettBarn} key={ettBarn.fodselsnummer} />);
+    const barnListe = barn.map((ettBarn, index) => <EnkeltBarn barn={ettBarn} key={index} />);
```

Comment:
Hvorfor er dette endret på? Index som key er noe vi burde la være å bruke da det er mulig.

Reply:
Med gamle kode, fikk jeg feilmelding i konsollen som:-
index.js:2178 Warning: Encountered two children with the same key, `10108000398`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

Reply:
haha, er sikkert mock dataen som gir feilen. barna har sikkert samme fnr :P

Reply:
Det bør vi nok da løse opp i. 😛